src/components/NavBar/NavIcon.css.ts

```diff
@@ -15,6 +15,7 @@ export const navIcon = style([
     cursor: 'pointer',
     borderRadius: '8',
     transition: '250',
+    margin: '4',
```

Comment:
`margin-right` would be better for this purpose. Having the top-margin un-aligns the menu button
![Screen Shot 2023-05-23 at 10 26 05 ](https://github.com/Uniswap/interface/assets/11512321/f4864dc1-7c75-4caf-b9b6-57c39e28e251)

Reply:
On it